2022-02-14T20:53:15 INFO: Loading datasets
2022-02-14T20:53:21 INFO: Fetching fastText model for OCR processing
2022-02-14T20:53:21 WARNING: /home/ubuntu/hzy/pythia/pythia/datasets/processors.py:402: UserWarning: No model file present at /home/ubuntu/hzy/pythia/models/.vector_cache/wiki.en.bin.
  warnings.warn("No model file present at {}.".format(model_file))

2022-02-14T20:53:21 INFO: Downloading FastText bin
2022-02-14T21:16:45 INFO: fastText bin downloaded at /home/ubuntu/hzy/pythia/pythia/.vector_cache/wiki.en.bin.
2022-02-14T21:16:45 INFO: Loading fasttext model now from /home/ubuntu/hzy/pythia/pythia/.vector_cache/wiki.en.bin
2022-02-14T21:17:07 INFO: Finished loading fasttext model
2022-02-14T21:17:21 ERROR: [Errno 2] No such file or directory: '/raid/lixiangpeng/dataset/textvqa/m4c_data/detectron/fc6/fc7_w.pkl'
